src/components/Embedded/__snapshots__/Embedded.test.js.snap

```diff
@@ -14,35 +14,35 @@ exports[`provides an API to communicate with an embedded publication 1`] = `
     className="links"
   >
     <a
-      href="/on-the-murder-of-eratosthenes-1-50/5"
+      href="https://www.example.com/on-the-murder-of-eratosthenes-1-50/5"
```

Comment:
Most of the diffs in this PR are due to a change in the test setup: I set `process.env.PUBLIC_URL` to `"https://www.example.com"` because the `cross-fetch` library requires full URLs.